facia/app/controllers/FaciaController.scala

```diff
@@ -175,7 +176,11 @@ trait FaciaController
     } else result
 
   import PressedPage.pressedPageFormat
-  private[controllers] def renderFrontPressResult(path: String)(implicit request: RequestHeader) = {
+  private[controllers] def renderFrontPressResult(path: String)(implicit request: RequestHeader): Future[Result] = {
+    val participatingInTest = ActiveExperiments.isParticipating(EuropeNetworkFront)
+    if (!participatingInTest && path == "europe") {
```

Comment:
```suggestion
    if (path == "europe" && !participatingInTest) {
```

Just a suggestion but my brains find this order easier to read